### ISSUE TITLE
lintr comment bot seems to fail

### ISSUE THREAD

**Issue by zachmayer** [jimhester/lintr#133]
The lintr comment bot tried and failed to comment on one of my PRs:
https://travis-ci.org/zachmayer/caretEnsemble/jobs/106112153

``` R
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  12: print.lints(lints)
  13: github_comment(lint_output, info, ...)
  14: message(httr::http_condition(response, "error", message = httr::content(response, 
         as = "text")))
  15: httr::http_condition(response, "error", message = httr::content(response, as = "text"))

  testthat results ================================================================
  OK: 629 SKIPPED: 2 FAILED: 1
  1. Error: Code Lint 

  Error: testthat unit tests failed
  In addition: There were 50 or more warnings (use warnings() to see the first 50)
  Execution halted
```

**Comment by zachmayer** [comment]
I was using an old version of lintr.  When I updated the package version I was using on travis, it was fixed.

**Comment by jimhester** [comment]
Great, thanks for the update!
